app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -208,6 +227,24 @@ class ExoPlayerProfile(
 			}
 			// Audio channel profile
 			add(maxAudioChannelsCodecProfile(channels = if (downMixAudio) 2 else 8))
+			// Add maximum audio channel profiles for SPDIF if enableSPDIF is true
+			if (enableSPDIF) {
+				arrayOf(
+					Codec.Audio.AAC,
+					Codec.Audio.AAC_LATM,
+					Codec.Audio.ALAC,
+					Codec.Audio.PCM_ALAW,
+					Codec.Audio.PCM_MULAW,
+					Codec.Audio.PCM_S16LE,
+					Codec.Audio.PCM_S20LE,
+					Codec.Audio.PCM_S24LE,
+					Codec.Audio.OPUS,
+					Codec.Audio.FLAC,
+					Codec.Audio.VORBIS
+				).forEach { codec ->
```

Comment:
It should be possible to use a comma delimited list for codec in a single codec profile instead of adding separate profiles.